docs/examples.md

```diff
@@ -9,103 +9,115 @@ bodyclass: examples
 Every tutorial here comes with step-by-step code explanation and is easy enough even for beginner JavaScript developers.
 
 
-{% include tutorial_link.html
+{% raw %}
```

Comment:
These changes look unrelated to the actual topic of the PR. Please only change things relevant to the thing your trying to fix.